tests/e2e/specs/admin/amp-options.js

```diff
@@ -137,6 +137,30 @@ describe( 'Saving', () => {
 } );
 
 describe( 'AMP settings screen Review panel', () => {
+	let testPost;
+
+	beforeAll( async () => {
+		await visitAdminPage( 'admin.php', 'page=amp-options' );
+
+		testPost = await page.evaluate( () => wp.apiFetch( {
+			path: '/wp/v2/posts',
+			method: 'POST',
+			data: { title: 'Test Post', status: 'publish' },
+		} ) );
+	} );
+
+	afterAll( async () => {
+		await visitAdminPage( 'admin.php', 'page=amp-options' );
+
+		if ( testPost.id ) {
```

Comment:
Minor thing: shouldn't this be:

```suggestion
		if ( testPost ) {
```

If `testPost` was not set, then this would be an error due to an undefined index?

Reply:
That's a good catch. I think we could actually go a step further like:

```suggestion
		if ( testPost?.id ) {
```

Reply:
Yes, good.